tests/unit/customizations/configure/test_sso.py

```diff
@@ -256,6 +257,19 @@ def test_basic_configure_sso_flow(self):
         self.sso_stub.assert_no_pending_responses()
         self.assert_config_updates()
 
+    def test_single_account_single_role_flow_no_browser(self):
+        self._add_prompt_responses()
+        self._add_simple_single_item_responses()
+        with self.sso_stub:
+            self.configure_sso(
+                args=['--no-browser'],
+                parsed_globals=self.global_args,
+            )
```

Comment:
I wonder if there is any additional assertion we can make here to make sure that the browser does not open or we use the expected handler?